jsapp/js/components/activity/formActivity.scss

```diff
@@ -0,0 +1,91 @@
+@use 'scss/sizes';
+@use 'scss/mixins';
+@use 'scss/breakpoints';
+@use 'scss/colors';
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  background-color: colors.$kobo-white !important;
+
+  .header {
+    width: 100%;
+    display: flex;
+    gap: sizes.$x8;
```

Comment:
That `scss/sizes` is deprecated and I'm going to remove it next few days. We should use regular `px` values going forward :) So please change this file here to `px`, and I will have one file less to fix :P